components/PostArticle.vue

```diff
@@ -1,62 +1,52 @@
 <template>
-	<article>
-		<h1 v-html="title" class="title" />
-		<img v-if="picture" :src="picture" class="news-item-img">
-		<WeCountLogo v-else class="news-item-img" />
-		<div v-html="content" class="api-content" />
-		<div v-for="(t, index) in tags" :key="index">
-			<nuxt-link :to="{ path: '/tag', query: { s: t }}">
-				{{ t }}
-			</nuxt-link>
-		</div>
+	<article class="post-article">
+		<h1 v-html="postToRender.title" class="title" />
+
+		<div v-html="postToRender.author" class="author" />
+
+		<div v-html="postToRender.date" class="date" />
+
+		<div v-html="postToRender.content" class="api-content" />
+
+		<section class="tags-info">
```

Comment:
@greatislander should we use article instead of section?

Reply:
> should we use article instead of section?
It probably makes sense to discuss which tag should be used here when we revisit the semantic structure of site pages.

Reply:
Okay let's bring this up in today's call

Reply:
In today's call, we agreed on using `<section>`.